partials/CollapsingRightSidebarContent.txt

```diff
@@ -52,6 +52,14 @@
     <a href="https://chocolatey.org/events/chocolatey-product-spotlight" rel="noreferrer" target="_blank" class="btn btn-primary mt-2 mx-1">Learn More</a>
 </div>
 <hr />
+<div class="text-center">
+    <a href="https://chocolatey.org/events/chocolatey-coding-livestream" rel="noreferrer" target="_blank">
+        <img class="border rounded mb-3" src="https://img.chocolatey.org/events/01-21-2.jpg" alt="Chocolatey Coding Livestream" />
+    </a>
+    <p class="text-start">Join us for the Chocolatey Coding Livestream, where members of our team dive into the heart of open source development by coding live on various Chocolatey projects. Tune in to witness real-time coding, ask questions, and gain insights into the world of package management. Don’t miss this opportunity to engage with our team and contribute to the future of Chocolatey!</p>

```

Comment:
@st3phhays I am going to push a slight change to this PR, as there is a gremlin character here, which I believe is causing this artifact on the site: 
<img width="402" alt="image" src="https://github.com/chocolatey/choco-theme/assets/1271146/2524bbfb-e71e-4097-bae0-c94d3b4af6b7">


```suggestion
    <p class="text-start">Join us for the Chocolatey Coding Livestream, where members of our team dive into the heart of open source development by coding live on various Chocolatey projects. Tune in to witness real-time coding, ask questions, and gain insights into the world of package management. Don't miss this opportunity to engage with our team and contribute to the future of Chocolatey!</p>
```